game/dota_addons/legion_td/scripts/vscripts/abilities/modifier_enchantress_untouchable_lua.lua

```diff
@@ -0,0 +1,55 @@
+modifier_enchantress_untouchable_lua = class({})
```

Comment:
This file should probably be in `vscripts/abilities/waveunits/` like `enchantress_untouchable_lua.lua`.